### PR TITLE
Clean up styles

### PR DESCRIPTION
**Pull request by sashotoster**
This PR contains a minor cleanup for index.styl file (removing duplicated rules, rules that will never be applied etc) without affecting current functionality.

### CODE REVIEW

**Comment by NikolayRys** [comment]
Looks great! I would only suggest to clean up a bit the commit history, so it's navigable better and doesn't have those merges. 
Best would be to have 3-5 commits for this amount of changes, but if you want simply squash it, then it's also fine.

File: source/index.styl [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
+
+    * {
+      visibility: inherit;
+    }
+  }
 }
 
 .likely {
-  // outline: 1px #f00 solid;
-  &__widget {
```
Comment (vitkarpov) [inline source/index.styl:34]: Why don't we need the same for &__widget anymore?

**Comment by sashotoster** [comment]
> Am I right that the built file should've not been changed except, probably, the order of rules? I mean we're trying to refactor styles without changing meaning, aren't we?

True, there should be no changes when it comes to applied rules (as in, everything is supposed to look exactly the same). However, part of this refactoring was aimed at reducing the size of resulting css file by removing rules that don't affect the styling for various reasons.

I tried to explain my reasoning in commit messages, here are some examples:
https://github.com/NikolayRys/Likely/pull/175/commits/92dfdbf5592b0ef864cfa00dc9759122dd64ef4a
https://github.com/NikolayRys/Likely/pull/175/commits/6620c6badf5a28c9b986aabdd0522c8cbcd422ca
https://github.com/NikolayRys/Likely/pull/175/commits/8d04b6745168217fdc4d51aa3fc2fa673c67e221

To sum up, no new rules are added, and existing rules are either relocated (which indeed affects the order in built file) or removed (from the built file as well).

**Comment by sashotoster** [comment]
> Looks great! I would only suggest to clean up a bit the commit history, so it's navigable better and doesn't have those merges.
> Best would be to have 3-5 commits for this amount of changes, but if you want simply squash it, then it's also fine.

Got it. I'd prefer not to squash it in order to keep the reasoning clear for removing / relocating certain rules, but for sure I'll take a look at what can be improved here.

File: source/index.styl (hunk 1 above)
Comment (sashotoster) [reply source/index.styl:34]: As for this particular block, `display: inline-block` part was removed as duplicate (it was already covered by `.likely > *` selector, and the other two (`position: relative;`, `white-space: nowrap;`) are still there, moved below in the code (line 32).

In general, I also tried not to duplicate rules that are going to be inherited from .likely parent, since we currently don't support having a likely_widget on its own (it's always wrapped in .likely).

Comment (vitkarpov) [reply source/index.styl:34]: Gotcha, thanks!

**Comment by NikolayRys** [comment]
@sashotoster awesome, thanks :) Please, run build once more to resolve the conflict and it'll be mergeable
